ow_lander/src/ow_lander/actions.py

```diff
@@ -207,6 +208,18 @@ def modify_joint_positions(self, goal):
       pos[goal.joint] = goal.angle
     return pos
 
+class ArmSetToolServer(ActionServerBase):
+
+  name          = 'ArmSetTool'
+  action_type   = owl_msgs.msg.ArmSetToolAction
+  goal_type     = owl_msgs.msg.ArmSetToolGoal
+  feedback_type = owl_msgs.msg.ArmSetToolFeedback
+  result_type   = owl_msgs.msg.ArmSetToolResult
+
```

Comment:
The action client blocks and does nothing because the action server is never started using `_start_server`. Since this is a dummy action, and does not use the arm interface, it does make sense that it does not inherit from `ArmActionMixin`, which would only needlessly initialize an arm and planning interface, but you still require a call to `_start_server` in this class's `__init__`. See other non-arm actions such as `LightSetIntensity` for an example. 

```suggestion
class ArmSetToolServer(ActionServerBase):

  name          = 'ArmSetTool'
  action_type   = owl_msgs.msg.ArmSetToolAction
  goal_type     = owl_msgs.msg.ArmSetToolGoal
  feedback_type = owl_msgs.msg.ArmSetToolFeedback
  result_type   = owl_msgs.msg.ArmSetToolResult
  
  def __init__(self, *args, **kwargs):
    super(ArmSetToolServer, self).__init__()
    self._start_server()

```

Reply:
Thanks, that was it!  I'll let you know when this PR is ready for review - or when I have another implementation question.